nodes/nodes/nodes-nodes-working.adoc

```diff
@@ -33,6 +33,8 @@ include::modules/nodes-nodes-kernel-arguments.adoc[leveloffset=+1]
 ifdef::openshift-webscale[]
 include::modules/nodes-nodes-rtkernel-arguments.adoc[leveloffset=+1]
 endif::openshift-webscale[]
+include::modules/nodes-nodes-cgroups-2.adoc[leveloffset=+1]
+
```

Comment:
I'm not sure if this is a concern, but I did notice that this causes multiple Additional Resource sections stacked together: https://deploy-preview-39619--osdocs.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-working.html#additional-resources

Reply:
I agree, this looks a bit odd. Is there a way to get around this?

Reply:
Well done! 
lgtm!